app/views/setup/worker_bootstrap.html.slim

```diff
@@ -12,19 +12,19 @@ h1 Bootstrap your Container as a Service Platform
 p
   | In order to complete the installation, it is necessary to bootstrap a few additional nodes, those will be the Kubernetes Master and Workers.
   |  This process leverages AutoYaST and is (almost) fully automated.
-  |  In case you are not familiar with it, you can find more information about AutoYaST in the 
-  a href="https://www.suse.com/documentation/sles-12/singlehtml/book_autoyast/book_autoyast.html" official documentation.
-  |  The automatic installation gets invoked by adding autoyast=&lt;URL_TO_PROFILE&gt; to the kernel parameter list.
+  |  In case you are not familiar with it, you can find more information about AutoYaST in the
+  a href="https://www.suse.com/documentation/sles-12/singlehtml/book_autoyast/book_autoyast.html"  official documentation.
```

Comment:
Otherwise, the link appears next to the regular text :(

It's the first time I use slim, but this is the only way I found to show everything correctly.